Application/EdFi.Ods.Api/Security/AuthorizationStrategies/Relationships/Filters/RelationshipsAuthorizationFilters.cs

```diff
@@ -30,30 +30,21 @@ public static class RelationshipsAuthorizationFilters
         private static readonly Lazy<FilterApplicationDetails> _stateEducationAgencyIdToStateEducationAgencyId =
             new Lazy<FilterApplicationDetails>(() => CreateClaimValuePropertyFilter("StateEducationAgencyId"));
 
-        private static FilterApplicationDetails CreateClaimValuePropertyFilter(string propertyName)
-        {
-            return new FilterApplicationDetails(
-                $"{propertyName}To{propertyName}",
-                $"{propertyName} IN (:{propertyName})",
-                $"{{currentAlias}}.{propertyName} IN (:{propertyName})",
-                (c, w, p, jt) => w.ApplyPropertyFilters(p, propertyName),
-                (t, p) => p.HasPropertyNamed(propertyName));
-        }
-
         private static readonly Lazy<FilterApplicationDetails> _localEducationAgencyIdToStudentUSI
             = new Lazy<FilterApplicationDetails>(
                 () =>
                     new FilterApplicationDetails(
                         "LocalEducationAgencyIdToStudentUSI",
                         @"StudentUSI IN (
                             SELECT {newAlias1}.StudentUSI 
-                            FROM auth.LocalEducationAgencyIdToStudentUSI {newAlias1} 
-                            WHERE {newAlias1}.LocalEducationAgencyId IN (:LocalEducationAgencyId))",
+                            FROM auth.StudentUSIToEducationOrganizationId {newAlias1} 
+                            WHERE {newAlias1}.SourceEducationOrganizationId IN (:SourceEducationOrganizationId))",
                         @"{currentAlias}.StudentUSI IN (
                             SELECT {newAlias1}.StudentUSI 
-                            FROM " + "auth_LocalEducationAgencyIdToStudentUSI".GetFullNameForView() + @" {newAlias1} 
-                            WHERE {newAlias1}.LocalEducationAgencyId IN (:LocalEducationAgencyId))",
-                        (c, w, p, jt) => c.ApplyJoinFilter(w, p, "LocalEducationAgencyIdToStudentUSI", "StudentUSI", "LocalEducationAgencyId", jt),
+                            FROM " + "auth_StudentUSIToEducationOrganizationId".GetFullNameForView() + @" {newAlias1} 
+                            WHERE {newAlias1}.SourceEducationOrganizationId IN (:LocalEducationAgencyId))",
+                        (c, w, p, jt) => c.ApplyJoinFilter(
+                            w, p, "StudentUSIToEducationOrganizationId", "StudentUSI", "SourceEducationOrganizationId", jt),
```

Comment:
This is what's causing issues when you have multiple filters triggered (such as from having more than one ed org type claim).  This view name here is also used to generate the alias, and it assumes you will only ever use each view once (since that's how it used to work).  You probably need to adjust this ApplyJoinFilter method to accept an explicit alias parameter, so we can separate the alias (which has to be unique over the entire query) from the view name (which is used in the generated hql/sql).

Reply:
/data/v3/ed-fi/StudentSectionAssociations/?schoolId={{known:schoolId}}
/data/v3/ed-fi/students?schoolId={{known:schoolId3}}
I tested this both resource using MultipleEdorg types and working fine in my local 
{
      "Email": "multipleEdorg4@ed-fi.org",
      "VendorName": "multipleEdorg4",
      "Applications": [
        {
          "ApplicationName": "ODS/API",
          "ClaimSetName": "SIS Vendor",
          "ApiClients": [
            {
              "ApiClientName": "MultipleEdorg_255901999_255901",
              "LocalEducationOrganizations": [ 255901999, 255901 ]
            }
          ]
        }
      ],
      "NamespacePrefixes": [ "uri://ed-fi.org" ]
    }